Yank/tests/test_repex.py

```diff
@@ -324,7 +324,7 @@ def decompact_state_variable(variable):
                 unsampled_serialized.append(unsampled_dict)
 
             # Two of the three ThermodynamicStates are compatible.
-            assert isinstance(states_serialized[0]['standard_system'], str)
+            assert 'standard_system' in states_serialized[0]
```

Comment:
This test will always be true. This will only check that the key is in the dict. When no system is returned, the `None` object occupies the value, but the `in <dict>` operator is only over the keys.